website/content/en/docs/contribution-guidelines/releasing.md

```diff
@@ -4,391 +4,275 @@ linkTitle: Releasing
-
-#### Cherry-picking
+- `changelog/generated/v1.3.0.md`: commit changes (created by changelog generation).
+- `changelog/fragments/*`: commit deleted fragment files (deleted by changelog generation).
+- `website/content/en/docs/upgrading-sdk-version/v1.3.0.md`: commit changes (created by changelog generation).
+- `website/config.toml`: commit changes (modified by release script).
 
-Once a minor release is complete, bug fixes can be merged into the release branch for the next patch release.
-Fixes can be added automatically by posting a `/cherry-pick v1.3.x` comment in the `master` PR, or manually by running:
+Commit these changes and push:
 
 ```sh
-$ git checkout v1.3.x
-$ git checkout -b cherrypick/some-bug
-$ git cherry-pick "$GIT_COMMIT_HASH" # Hash of the merge commit to master.
-$ git push upstream cherrypick/some-bug
+git add --all
+git commit -m "Release v1.3.0"
+git push -u origin release-v1.3.0
 ```
 
-Create and merge a PR from your branch to `v1.3.x`.
-
-## GitHub release information
-
-### Locking down branches
-
-Once a release PR has been made and all tests pass, the SDK's `master` branch, or [release branch](#release-branches) for patch releases,
-should be locked so commits cannot happen between the release PR and release tag push. To lock down a branch:
-
-1. Go to `Settings -> Branches` in the SDK repo.
-1. Under `Branch protection rules`, click `Edit` on the `master` or release branches rule.
-1. In section `Protect matching branches` of the `Rule settings` box, increase the number of required approving reviewers to its maximum allowed value.
-
-Now only administrators (maintainers) should be able to force merge PRs. Make sure everyone in the relevant Slack channel is aware of the release so they do not force merge by accident.
-
-Unlock `master` or release branch after the release has completed (after step 6 is complete) by changing the number of required approving reviewers back to 1.
-
-### Releasing
-
-The GitHub [`Releases` tab][release-page] in the operator-sdk repo is where all SDK releases live.
-To create a GitHub release see the [releasing binaries section](#9-releasing-binaries-signatures-and-release-notes).
+### 2. Lock down branch `master`
 
-#### Release notes
-
-GitHub release notes should thoroughly describe changes made to code, documentation, and design of the SDK. PR links should be included wherever possible.
-
-The following sections, often directly copied from our [changelog][doc-changelog], are used as release notes:
-
-```Markdown
-[Version as title, ex. v1.2.3]
-
-### Added
-- [Short description of feature added] (#PR)
-...
-
-### Changed
-- [Short description of change made] (#PR)
-...
-
-### Deprecated
-- [Short description of feature deprecated] (#PR)
-...
+Lock down the `master` branch to prevent further commits before the release completes.
+See [this section](#locking-down-branches) for steps to do so.
```

Comment:
Linking to other sections seems like a good DRY principle, but it makes following the release doc really hard because you end up jumping from section to section. I would rather put the locking instructions directly in this section.